src/_pytest/helpconfig.py

```diff
@@ -91,6 +91,14 @@ def pytest_addoption(parser: Parser) -> None:
         action="append",
         help='override ini option with "option=value" style, e.g. `-o xfail_strict=True -o cache_dir=cache`.',
     )
+    group._addoption(
+        "-a",
+        "--append-ini",
+        dest="append_ini",
+        action="append",
+        help='append to an `args`, `linelist`, or `pathlist` type ini option with "option=value" style, e.g. '
```

Comment:
Here too I suggest not mentioning the types because they are rather obscure. Maybe use the term "multi-valued ini option"?

Reply:
I have the same comment for this as I left here: https://github.com/pytest-dev/pytest/pull/7391#discussion_r449662606

I can go either way, but I feel like listing out the types is consistent with the current way we document things